### PR TITLE
3CBF weapon integration into RHS factions

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
added a case that adds a couple appropriate weapons in the case that an pure RHS faction is used while 3CB Factions is loaded.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_AFRF_Arid.sqf [hunk 1]
```diff
@@ -390,6 +390,23 @@ _pilotLoadoutData set ["uniforms", ["rhs_uniform_df15_tan"]];
 _pilotLoadoutData set ["vests", ["rhs_vest_commander"]];
 _pilotLoadoutData set ["helmets", ["rhs_zsh7a_mike_alt", "rhs_zsh7a_mike", "rhs_zsh7a_mike_green", "rhs_zsh7a_mike_green_alt"]];
 
+if (isClass (configFile >> "CfgPatches" >> "UK3CB_Factions_Vehicles_SUV")) then {
+    (_militaryLoadoutData get "marksmanRifles") append [
+    ["UK3CB_SVD_OLD", "", "", "rhs_acc_pso1m21", ["rhs_10Rnd_762x54mmR_7N1"], [], ""]
+    ];
```
Comment (rautamiekka) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_AFRF_Arid.sqf:396]: All of these lines ain't exactly very readable like this.

**Review by rautamiekka — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_HIDF.sqf [hunk 2]
```diff
@@ -336,6 +336,34 @@ _pilotLoadoutData set ["vests", ["rhsgref_TacVest_ERDL"]];
...
+    ["UK3CB_M16A2_UGL", "", "", "", ["rhs_mag_20Rnd_556x45_M193_Stanag", "rhs_mag_20Rnd_556x45_M193_Stanag", "rhs_mag_20Rnd_556x45_M196_Stanag_Tracer_Red"], ["rhs_mag_M441_HE", "rhs_mag_M441_HE", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""],
+    ["UK3CB_M16A3_UGL", "", "", "", ["rhs_mag_20Rnd_556x45_M193_Stanag", "rhs_mag_20Rnd_556x45_M193_Stanag", "rhs_mag_20Rnd_556x45_M196_Stanag_Tracer_Red"], ["rhs_mag_M441_HE", "rhs_mag_M441_HE", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""]
+    ];
+    (_militiaLoadoutData get "machineGuns") append [
+    ["UK3CB_M60", "", "", "", ["UK3CB_M60_100rnd_762x51", "UK3CB_M60_100rnd_762x51_R", "UK3CB_M60_100rnd_762x51_RM", "UK3CB_M60_100rnd_762x51_RT"], [], ""]
+    ];
+
+    (_policeLoadoutData get "rifles") append [
+    "uk3cb_enfield_l8",
+    ["uk3cb_sks_01", "", "", "", ["uk3cb_10rnd_magazine_sks", "uk3cb_10rnd_magazine_sks_G", "uk3cb_10rnd_magazine_sks_GT"], [], ""]
```
Comment (Play3rgg) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_HIDF.sqf:364]: sks on HIDF?

File: A3A/addons/core/Templates/Templates/RHS/RHS_AI_TLA.sqf [hunk 3]
```diff
@@ -346,6 +346,55 @@ _pilotLoadoutData set ["vests", ["rhsgref_TacVest_ERDL"]];
...
+    ["UK3CB_RPD", "", "", "", ["UK3CB_RPD_100rnd_762x39", "UK3CB_RPD_100rnd_762x39_G", "UK3CB_RPD_100rnd_762x39_GM", "UK3CB_RPD_100rnd_762x39_GT"], [], ""],
+    ["UK3CB_RPK", "", "", "", ["UK3CB_RPK_40rnd_762x39", "UK3CB_RPK_40rnd_762x39_G", "UK3CB_RPK_40rnd_762x39_GM", "UK3CB_RPK_40rnd_762x39_GT"], [], ""],
+    ["UK3CB_M60", "", "", "", ["UK3CB_M60_100rnd_762x51", "UK3CB_M60_100rnd_762x51_R", "UK3CB_M60_100rnd_762x51_RM", "UK3CB_M60_100rnd_762x51_RT"], [], ""]
+    ];
+    (_militaryLoadoutData get "marksmanRifles") append [
+    ["UK3CB_SVD_OLD", "", "", "rhs_acc_pso1m21", ["rhs_10Rnd_762x54mmR_7N1"], [], ""]
+    ];
+    (_militaryLoadoutData get "sniperRifles") append [
+    ["UK3CB_SVD_OLD", "", "", "rhs_acc_pso1m21", ["rhs_10Rnd_762x54mmR_7N1"], [], ""],
+    ["UK3CB_SVD_OLD", "", "", "rhs_acc_1pn93_1", ["rhs_10Rnd_762x54mmR_7N1"], [], ""]
```
Comment (Play3rgg) [inline A3A/addons/core/Templates/Templates/RHS/RHS_AI_TLA.sqf:369]: lol

**Review by Play3rgg — CHANGES_REQUESTED** [review]
looks fine for the most part

**Review by Play3rgg — APPROVED** [review]
(I still think M193/M196 on RHS USAF is a bit silly since the normal RHS ones get M855 and only the 3cbf ones get the worse ammo
more variety ig)

LGTM

**Pull request merged**